pcsx2-qt/QtUtils.cpp

```diff
@@ -139,6 +140,95 @@ namespace QtUtils
 		ResizeColumnsForView(view, widths);
 	}
 
+
+	static int DPRScale(int size, qreal dpr)
+	{
+		return static_cast<int>(static_cast<qreal>(size) * dpr);
+	}
+
+	static int DPRUnscale(int size, qreal dpr)
+	{
+		return static_cast<int>(static_cast<qreal>(size) / dpr);
+	}
+
+	void resizeAndPadPixmap(QPixmap* pm, int expected_width, int expected_height, qreal dpr, const std::string& scaling, const float opacity)
```

Comment:
Can `scaling` be changed to an enum?
Something similar to the enum the BPM svg loader uses would work https://github.com/PCSX2/pcsx2/blob/07bc2fa452f61aa67e813f1ac14b9759fbdc56be/pcsx2/ImGui/ImGuiFullscreen.h#L108-L114